packages/react-workspace/react-client-app/src/components/App.tsx

```diff
@@ -299,7 +801,7 @@ export function App(): JSX.Element {
                         <div className="container-label">Select</div>
                         <NimbleSelect filter-mode="standard" appearance="underline">
                             Underline Select
-                            <NimbleListOption hidden selected disabled>Select an option</NimbleListOption>
+                            <NimbleListOption hidden disabled>Select an option</NimbleListOption>
```

Comment:
What's the rationale for removing `selected` from these list options? The Angular app keeps it and this state of hidden + selected + disabled is [documented as the best approach for placeholder values](https://nimble.ni.dev/storybook/?path=/docs/components-select--docs#placeholder) so ideally we'd keep using that approach here too.

https://github.com/ni/nimble/blob/4e353887ac1607dd2a05c0544e2b6189dd90c03c/packages/angular-workspace/example-client-app/src/app/customapp/customapp.component.html#L161

Reply:
Also added a note in storybook